comicapi/filenameparser.py

```diff
@@ -89,7 +89,7 @@ def get_issue_number(self, filename):
             # is the series name followed by issue
             filename = re.sub(r"--.*", self.repl, filename)
 
-        elif "__" in filename:
+        elif "__" in filename and not re.search('\\[__\\d+__\\]', filename):
```

Comment:
```suggestion
        elif "__" in filename and not re.search(r"\[__\d+__\]", filename):
```

Python has first-class support for regular expressions